NetKVM/wlh/ParaNdis6_Oid.cpp

```diff
@@ -562,6 +562,11 @@ static NDIS_STATUS ParaNdis_OidQuery(PARANDIS_ADAPTER *pContext, tOidDesc *pOid)
     {
         status = ParaNdis_OidQueryCopy(pOid, pInfo, ulSize, bFreeInfo);
     }
+    if (status == NDIS_STATUS_SUCCESS && pOid->Oid == OID_GEN_STATISTICS)
```

Comment:
General question: do we have other gaps in statistics?

Reply:
I hope we do not. Other thing is that on some VF drivers the statistics might be buggy. I see it on one of Intel drivers ((